docs/web-sdk.md

```diff
@@ -61,6 +61,29 @@ And then loaded in your JavaScript and bundled as you like:
 import '@exmg/livery';
 ```
 
+#### Using with Next.js
+Next.js makes use of server side rendering, which the Livery Web SDK does not currently support. However, users of Next.js can still use the SDK by making use of Next.js’s support for [dynamic imports](https://nextjs.org/docs/advanced-features/dynamic-import), which ensure the module being imported is evaluated at runtime on the client. The code below illustrates basic usage with Next.js. 
```

Comment:
You reference dynamic imports here but don't use them below; perhaps we should point to useEffect() documentation for client-side only execution? Though I have to admit I can't find that.

Reply:
Have I misunderstood dynamic imports? I was under the impression that they were merely `import( )` statements that reside in the flow of code rather than only at the top level of the module.

Reply:
That's true, but this page only seems to describe use of dynamic imports wrapped in the nextjs `dynamic()` function which is not used in this case. I'd prefer to see a link to documentation about using `useEffect()` for this purpose, but that does not seem to exist. Perhaps you could link to this page (https://nextjs.org/docs/basic-features/layouts#data-fetching) for `useEffect()` instead or, if in addition to the link to the dynamic-import page describe why you're not using `dynamic()` in this case for the dynamic import?

Reply:
I think it might be good (from an SEO perspective if nothing else) for us to provide some documentation about using standard web components with Next.js and React, but that also sounds like an extra distraction we don't need right now :(

I'll add a note to the effect "Note, many of the examples that Next.js provides use a statement of the form `dynamic( import(...) )`. The `dynamic()` function is used for importing React components and is not used for importing `@exmg/livery`"